### PR TITLE
Fixes 3145 - Adds an action-needsmoderation label to anonymous issue

### PR DESCRIPTION
**Pull request by karlcow**
This PR fixes issue #3145

## Proposed PR background

This is adding the label necessary for triager to filter out the issues which have not been yet been accepted. 
It was necessary to modify the webhook so that it will not crush the existing issue labels.

### CODE REVIEW

**Comment by karlcow** [comment]
ok now that #3150 is fixed (#3156 merged)
I will rebase this and @miketaylr can review eventually.

File: webcompat/issues.py [hunk 1]
```diff
@@ -57,7 +57,10 @@ def report_public_issue(form):
     Returns a requests.Response object.
     """
     path = 'repos/{0}'.format(REPO_URI)
-    return proxy_request('post', path, data=json.dumps(unmoderated_issue()))
+    public_data = unmoderated_issue()
+    # We add action-needsmoderation label, so reviewers can filter out
+    public_data['labels'] = ['action-needsmoderation']
```
Comment (miketaylr) [inline webcompat/issues.py]: I think eventually we will want to update https://github.com/webcompat/webcompat.com/blob/d56cd783366d725bcf2cf560723a590faf41e08f/webcompat/static/js/lib/models/label-list.js#L27, but this is fine for now -- the labels will appear on GitHub.

Comment (miketaylr) [reply webcompat/issues.py]: Filed https://github.com/webcompat/webcompat.com/issues/3161 as a follow up for someone. Not urgent.

**Comment by karlcow** [comment]
Ah interesting fail after the merg.

```
======================================================================
FAIL: Test that we get a moderation template back from a public issue report.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/karl/code/webcompat.com/env/lib/python3.7/site-packages/mock/mock.py", line 1330, in patched
    return func(*args, **keywargs)
  File "/Users/karl/code/webcompat.com/tests/unit/test_issues.py", line 73, in test_report_public_issue_returns_moderation_template
    mockpost.assert_called_with(ANY, data=data, headers=ANY, params=ANY)
  File "/Users/karl/code/webcompat.com/env/lib/python3.7/site-packages/mock/mock.py", line 944, in assert_called_with
    six.raise_from(AssertionError(_error_message(cause)), cause)
  File "<string>", line 3, in raise_from
AssertionError: expected call not found.
Expected: post(<ANY>, data='{"title": "In the moderation queue.", "body": "<p>This issue has been put in the moderation queue. A human\\nwill review if the message meets our current\\n<a href=\\"https://www.mozilla.org/en-US/about/legal/acceptable-use/\\">\\nacceptable use</a> guidelines.\\nIt will probably take a couple of days depending on the backlog.\\nOnce it has been reviewed, the content will be either made public\\nor deleted.</p>"}', headers=<ANY>, params=<ANY>)
Actual: post('https://api.github.com/repos/webcompat/webcompat-tests/issues', data='{"title": "In the moderation queue.", "body": "<p>This issue has been put in the moderation queue. A human\\nwill review if the message meets our current\\n<a href=\\"https://www.mozilla.org/en-US/about/legal/acceptable-use/\\">\\nacceptable use</a> guidelines.\\nIt will probably take a couple of days depending on the backlog.\\nOnce it has been reviewed, the content will be either made public\\nor deleted.</p>", "labels": ["action-needsmoderation"]}', headers={'Authorization': 'token 7d9fb710fbc6050570e64d06593b831e1ae4206f', 'User-Agent': 'webcompat/webcompat-bot'}, params=None)

----------------------------------------------------------------------
```

**Comment by miketaylr** [comment]
> Ah interesting fail after the merg.

I guess it's sad because it didn't expect there to be a `labels`? (the UA thing is interesting too)

**Comment by karlcow** [comment]
ok nailed the test failures.

**Comment by karlcow** [comment]
rha failure :D

**Comment by karlcow** [comment]
wait wait :) did I type too quick

**Comment by karlcow** [comment]
ok this time we should be good. Sorry about that. As soon there is the green light @miketaylr could you check https://github.com/webcompat/webcompat.com/pull/3159/commits/8c3caa4d3a8e125859549322c35eaa5301682c4a

**Comment by miketaylr** [comment]
lgtm, thanks!

**Comment by karlcow** [comment]
Ok now we have one last bit to put in place. :) #3140 
yeah!

**Pull request merged**